5.findBugCommit.js

```diff
@@ -21,11 +21,10 @@
 //  {
 //    commit: { timestamp: '2020-02-13T14:17:20.844Z', status: 'bad' },
 //    index: 78
-//  }  
+//  }
 
 const identifyFirstBadCommit = (commits) => {
-  
-}
-
+  if (!commits.length) return [];
+};
```

Comment:
Go back and practice binary search. Also pay attention to what functions are expecting to return. This function expects and object.